docs/_articles/ja/user-guide/perf-enhancements.md

```diff
@@ -14,4 +14,9 @@ To enable this beta feature:
 1. Select **File** > **Preferences** > **Settings** (Windows or Linux) or **Code** > **Preferences** > **Settings** (macOS).
```

Comment:
@lcampos, consider updating 
Title to Enhance Code Performance 
Intro lines to:
The changes we have made to enhance performance ensure that single file metadata deploys through a more efficient code path. Currently, these changes support only Apex and Visualforce metadata types. We’ll add support for more metadata types in future releases.
Beta Notice to:
NOTICE: The changes to enhance code performance are is currently in beta. If you find any bugs or have feedback, open a GitHub issue.

Reply:
Super tiny nit:
The changes we have made to enhance performance ensure that single file metadata deploys *run* through a more efficient code path. Currently, these changes only support Apex and Visualforce metadata types. We’ll add support for more metadata types in future releases.
Beta Notice to:
NOTICE: The changes to enhance code performance *are* currently in beta. If you find any bugs or have feedback, open a GitHub issue.